README.md

```diff
@@ -143,9 +143,9 @@ The following options can be passed to any command:
     --aws-sts-arn='': The AWS STS Role ARN which will assume role
     --cosi-driver-path='/var/lib/cosi/cosi.sock': unix socket path for COSI
     --cosi-sidecar-image='gcr.io/k8s-staging-sig-storage/objectstorage-sidecar/objectstorage-sidecar:v20221117-v0.1.0-22-g0e67387': The cosi side car container image
-    --db-image='centos/mongodb-36-centos7': The database container image
+    --db-image='"quay.io/sclorg/postgresql-15-c9s"': The database container image
     --db-storage-class='': The database volume storage class name
-    --db-type='postgres': The type of database container image (mongodb, postgres)
+    --db-type='postgres': The type of database container image (postgres)
```

Comment:
why do we need this flag? 
Can we remove it?

Reply:
I removed all references to DBType in the code. Left it in the CRS, but basically ignoring it